python/openai/openai_frontend/main.py

```diff
@@ -108,6 +108,12 @@ def parse_args():
         choices=["vllm", "tensorrtllm"],
         help="Manual override of Triton backend request format (inputs/output names) to use for inference",
     )
+    triton_group.add_argument(
+        "--model-and-lora-name-separator",
```

Comment:
`--lora-separator` for conciseness?

Reply:
Sure, updated: [Rename --model-and-lora-name-separator to --lora-separator](https://github.com/triton-inference-server/server/pull/8038/commits/09a0868a2c1c000d1beee4f6fbfeb83090accd8c)